File: baekjoon/js/2941.js
const fs = require("fs");
const buf = fs.readFileSync(0, "utf-8").toString().trim();

let word = buf;

const croatiaAlphabet = ["c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="];

for (const alphabet of croatiaAlphabet) {
  word = word.replaceAll(alphabet, ".");
}

const result = word.length;

console.log(result);

File: baekjoon/rust/2941.rs
use std::io::stdin;

fn main() {
    let mut buf = String::new();
    stdin().read_line(&mut buf).unwrap();

    let mut word = buf.trim().to_string();

    let croatia_alphabets = ["c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="];

    for c in croatia_alphabets {
        word = word.replace(c, ".");
    }

    let result = word.len();

    print!("{}", result);
}
